src/univariate/continuous/normalcanon.jl

```diff
@@ -37,32 +40,32 @@ mean(d::NormalCanon) = d.μ
 median(d::NormalCanon) = mean(d)
 mode(d::NormalCanon) = mean(d)
 
-skewness(d::NormalCanon) = 0.0
-kurtosis(d::NormalCanon) = 0.0
+skewness{T<:Real}(d::NormalCanon{T}) = zero(T)
+kurtosis{T<:Real}(d::NormalCanon{T}) = zero(T)
 
-var(d::NormalCanon) = 1.0 / d.λ
+var(d::NormalCanon) = 1 / d.λ
 std(d::NormalCanon) = sqrt(var(d))
 
-entropy(d::NormalCanon) = 0.5 * (log2π + 1.0 - log(d.λ))
+entropy(d::NormalCanon) = (log2π + 1 - log(d.λ)) / 2
 
 
 #### Evaluation
 
-pdf(d::NormalCanon, x::Float64) = (sqrt(d.λ) / sqrt2π) * exp(-0.5 * d.λ * abs2(x - d.μ))
-logpdf(d::NormalCanon, x::Float64) = 0.5 * (log(d.λ) - log2π - d.λ * abs2(x - d.μ))
+pdf(d::NormalCanon, x::Real) = (sqrt(d.λ) / sqrt2π) * exp(-1/2* d.λ * abs2(x - d.μ))
```

Comment:
Yes. Sorry for being too brief. Divide with two instead of multiplying with `1/2==0.5`